specification/app/Microsoft.App/models/code-execution-file.tsp

```diff
@@ -0,0 +1,37 @@
+import "./common.tsp";
+using TypeSpec.Http;
+using TypeSpec.Rest;
+
+namespace Microsoft.App;
+
+@doc("Code execution file resource collection.")
+model CodeExecutionFileCollection {
+  @doc("The files uploaded.")
+  value: CodeExecutionFile[];
+}
+
+@doc("Code execution file resource.")
+@parentResource(SessionPool)
+@resource("files")
+model CodeExecutionFile {
```

Comment:
would you consider renaming it to `CodeInterpreterSessionResourceFile` ? 

Similarly file name and descriptions accordingly